pkg/urbit/vere/io/http.c

```diff
@@ -677,6 +677,8 @@ _http_start_respond(u3_hreq* req_u,
     if ( 0 == strncmp(hed_u->nam_c, "content-length", 14) ) {
       has_len_i = 1;
     }
+    // ignore connection specific headers for HTTP2
+    else if ( 0 == strncmp(hed_u->nam_c, "connection", 10) ) {}
```

Comment:
I think this header should only be ignored if the request used http2. You can check that with `0x200 <= rec_u-version`.